### DISCUSSION TITLE
BPA rule change for Format String Expressions

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#887] [Other Topics]
Is the following change (in bold) acceptable to change the BPA rules to require visible measures are formatted? It was failing because the Format String was null, but that is ok if the Format String Expression is filled out. Just want to make sure I haven't overlooked a potential issue. I just guessed at the object name.

IsVisible 
and string.IsNullOrWhitespace(FormatString) 
and **string.IsNullOrWhitespace(FormatStringExpression)** and (DataType = "Int64" or DataType = "DateTime" or DataType = "Double" or DataType = "Decimal")

**Comment by otykier** [comment]
Looks good! Should probably suggest this change over at https://github.com/microsoft/Analysis-Services/issues as well (this is the origin of the official set of BPA rules created by Michael Kovalsky).
